traffic_ops/traffic_ops_golang/vendor/golang.org/x/crypto/.gitattributes

```diff
@@ -0,0 +1,10 @@
+# Treat all files in this repo as binary, with no git magic updating
```

Comment:
I'd vote removing this file, too. I'm not enough of a Git Wizard to know if it's going to mess with our repo, but `.git*` files in subdirectories make me anxious it's going to mess with something. I don't think it'll hurt to delete it.

Reply:
will do